src/common.props

```diff
@@ -30,7 +30,6 @@
   <PropertyGroup>
     <Authors>Microsoft</Authors>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <PackageIcon>logo.png</PackageIcon>
```

Comment:
no logo? :(

Reply:
I unfortunately made this change a long time ago but hadn't yet pushed it, and now I can't remember exactly why I did this. I suspect I did it due to some warning, but I can't remember what the warning was and can't find any documentation explaining why having this would be a problem. I'll revert this change and revisit later.